frontend/src/component/segments/CreateSegment/CreateSegment.tsx

```diff
@@ -112,7 +115,8 @@ export const CreateSegment = ({ modal }: ICreateSegmentProps) => {
             >
                 <CreateButton
                     name='segment'
-                    permission={CREATE_SEGMENT}
+                    permission={[CREATE_SEGMENT, UPDATE_PROJECT_SEGMENT]}
+                    projectId={projectId}
```

Comment:
These 2 lines fix the bug on the UI, where the button was shown as being disabled.